src/frontend/screens/Settings/ProjectSettings/RemoteArchive/RemoveRemoteArchive.tsx

```diff
@@ -0,0 +1,129 @@
+import {useRemoveServerPeer} from '@comapeo/core-react';
+import {type NativeStackNavigationOptions} from '@react-navigation/native-stack';
+import * as Sentry from '@sentry/react-native';
+import {defineMessages, useIntl} from 'react-intl';
+import {StyleSheet, View} from 'react-native';
+import {UIActivityIndicator} from 'react-native-indicators';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+
+import {useActiveProject} from '../../../../contexts/ActiveProjectContext';
+import Error from '../../../../images/Error.svg';
+import {BottomSheetWrapper} from '../../../../sharedComponents/BottomSheetWrapper';
+import {
+  DestructiveButton,
+  SecondaryButton,
+} from '../../../../sharedComponents/Buttons';
+import {BodyText} from '../../../../sharedComponents/Text/BodyText';
+import {HeaderText} from '../../../../sharedComponents/Text/HeaderText';
+import {type NativeRootNavigationProps} from '../../../../sharedTypes/navigation';
+
+const m = defineMessages({
+  title: {
+    id: 'Settings.ProjectSettings.RemoteArchive.RemoveRemoteArchive.title',
+    defaultMessage: 'Remove {name} from project?',
+    description:
+      'Text for title that asks about confirming archive server removal.',
+  },
+  description: {
+    id: 'Settings.ProjectSettings.RemoteArchive.RemoveRemoteArchive.description',
+    defaultMessage:
+      'This will stop archiving new data for this project. Existing archived data will not be deleted.',
+    description: 'Explanation for consequences of removing archive server.',
+  },
+  removeArchive: {
+    id: 'Settings.ProjectSettings.RemoteArchive.RemoveRemoteArchive.removeArchive',
+    defaultMessage: 'Remove Archive',
+    description: 'Text for destructive action button.',
+  },
+  cancel: {
+    id: 'Settings.ProjectSettings.RemoteArchive.RemoveRemoteArchive.cancel',
+    defaultMessage: 'Cancel',
+    description: 'Text for cancel action button.',
+  },
+});
+
+export function RemoveRemoteArchive({
+  navigation,
+  route,
+}: NativeRootNavigationProps<'RemoveRemoteArchive'>) {
+  const {baseUrl, name, serverDeviceId} = route.params;
+
+  const {formatMessage: t} = useIntl();
+  const {projectId} = useActiveProject();
+
+  const removeServerPeer = useRemoveServerPeer({projectId});
+
+  return (
+    <BottomSheetWrapper>
+      <View style={styles.container}>
+        <View style={styles.infoContainer}>
+          <Error />
+          <HeaderText variant="header2" style={{textAlign: 'center'}}>
+            {t(m.title, {name: name || baseUrl})}
```

Comment:
technically the `name` can be undefined so figured it would be okay to display the base url if that's the case. open to other suggestions though